numba/core/typeconv/typeconv.py

```diff
@@ -3,18 +3,18 @@
     # Numba, if it fails to import, provide some feedback
     from numba.core.typeconv import _typeconv
 except ImportError as e:
-    from numba.core.errors import feedback_details as reportme
-    import sys
-    url = "https://numba.pydata.org/numba-doc/latest/developer/contributing.html"
+    base_url = "https://numba.readthedocs.io/en/stable"
+    dev_url = f"{base_url}/developer/contributing.html"
+    user_url = f"{base_url}/user/faq.html#numba-could-not-be-imported"
     dashes = '-' * 80
-    msg = ("Numba could not be imported.\nIf you are seeing this message and "
-           "are undertaking Numba development work, you may need to re-run:\n\n"
-           "python setup.py build_ext --inplace\n\n(Also, please check the "
```

Comment:
I would argue that including build instructions here may be a form of unneeded replication. I think it will suffice to just point folks at the development docs. Everyone will end up having their own way to build Numba and having multiple sources of instructions will only be confusing.

Reply:
They were here originally as an incompatible local source build was quite common (and maybe still is, we can't know because this message tells people what to do). Typically, a local source based development environment couldn't load this DSO because of python version mismatch or internal API changes due to development which leads to missing symbols. Perhaps this should go into the FAQ doc?

Reply:
Perhaps, yes.

Reply:
Done as part of: 9d4a5f0

The error message now reads:
```
ImportError: Numba could not be imported.
--------------------------------------------------------------------------------
If you are seeing this message and are undertaking Numba development work, you may need to rebuild Numba.
Please see the development set up guide:

https://numba.readthedocs.io/en/stable/developer/contributing.html.

--------------------------------------------------------------------------------
If you are not working on Numba development, the original error was: 'some problem'.
For help, please visit:

https://numba.readthedocs.io/en/stable/user/faq.html#numba-could-not-be-imported

```

Reply:
perfect!